xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentPassenger_version.xsd

```diff
@@ -201,6 +201,21 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Key issuing scheme under which facility is accessible.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="HandWashing" type="xsd:boolean" minOccurs="0">
```

Comment:
I would recommend adding sane default values.

Reply:
good point .. I'll do it

Reply:
done